Shared/UserAgent.swift

```diff
@@ -27,12 +27,16 @@ public class UserAgent {
      * Use this if you know that a value must have been computed before your
      * code runs, or you don't mind failure.
      */
-    public static func cachedUserAgent(checkiOSVersion checkiOSVersion: Bool = true) -> String? {
+    public static func cachedUserAgent(checkiOSVersion checkiOSVersion: Bool = true, checkFirefoxVersion: Bool = true) -> String? {
```

Comment:
Looking at the codebase, we don't actually pass `checkiOSVersion: false` anywhere. I'd probably just get rid of these arguments altogether, but up to you.